recipes/libxml2/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export CPPFLAGS="-I${PREFIX}/include"
+
+if [ $(uname) == Darwin ]; then
+  export OPTS="--disable-dependency-tracking --without-lzma"
```

Comment:
From the help info from `./configure`:

    --enable-dependency-tracking
                            do not reject slow dependency extractors
    --disable-dependency-tracking
                            speeds up one-time build

I think homebrew is just doing this to speed up their tests. We probably shouldn't set it for an actual build.